digitaltwins/samples/device/javascript/environmentalinterface.js

```diff
@@ -9,14 +9,14 @@ const Command = require('azure-iot-digitaltwins-device').Command;
 
 module.exports.EnvironmentalSensor = class EnvironmentalSensor extends BaseInterface {
   constructor(name, propertyCallback, commandCallback) {
-    super(name, 'dtmi:contoso:com:EnvironmentalSensor;1', propertyCallback, commandCallback);
+    super(name, 'dtmi:YOUR_COMPANY_NAME_HERE:EnvironmentalSensor;1', propertyCallback, commandCallback);
     this.temp = new Telemetry();
     this.humid = new Telemetry();
     this.state = new Property();
     this.blink = new Command();
-    this.turnOff = new Command();
```

Comment:
turnoff is not one word so i don't see how this follows the proper node.js camelcase naming conventions? Are we enforcing that all properties must be lowercase?

Reply:
This should be consistent with the DTDL  interface where the command name is defined as `turnon` lowercase.

Reply:
It is PnP who impose this case. not jscript naming rule here!